src/Compatibility/Microsoft.DotNet.PackageValidation/Microsoft.DotNet.PackageValidation.csproj

```diff
@@ -13,11 +13,13 @@
 
   <ItemGroup>
     <Compile Include="..\..\Tasks\Common\**\*.cs" LinkBase="Common" />
-    <PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildFrameworkVersion)" ExcludeAssets="Runtime" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildUtilitiesCoreVersion)" ExcludeAssets="Runtime" />
-    <PackageReference Include="NuGet.Frameworks" Version="$(NuGetBuildTasksPackageVersion)" ExcludeAssets="Runtime" />
-    <PackageReference Include="NuGet.Packaging" Version="$(NuGetBuildTasksPackageVersion)" ExcludeAssets="Runtime" />
-    <PackageReference Include="NuGet.Protocol" Version="$(NuGetBuildTasksPackageVersion)" ExcludeAssets="Runtime" />
+    <!-- We pin the versions of the assets that ship with the SDK in order to be able to run on older SDKs. 
+    Currently we support .NET 6 Preview5 or later. -->
+    <PackageReference Include="Microsoft.Build.Framework" Version="15.4.8" ExcludeAssets="Runtime" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.4.8" ExcludeAssets="Runtime" />
+    <PackageReference Include="NuGet.Frameworks" Version="6.0.0-preview.1.66" />
```

Comment:
Do we still need to pin NuGet version if we aren't excluding it from the package?

Reply:
Does MSBuild support loading assemblies listed in their assets.json that are older/newer than the ones in the running SDK?

Reply:
Give it a try :) I think that should be the current dotnet/runtime scenario.  The fact that they didn't force the downgrade to the CLI version of nuget leads me to believe they'd permit the local copy to load.  (Plus that's likely how things were working before we removed NuGet).